### PR TITLE
*: Fix ineffectual assignments

### PR DESCRIPTION
**Pull request by burntcarrot**
Related #690.

### CODE REVIEW

**Comment by burntcarrot** [comment]
The errors returned by `structcheck` are I guess false positives, as these fields are referenced by code below it.

**Comment by burntcarrot** [comment]
Added `nolint` to these fields, to ensure that the CI passes. Can you review the PR now?

File: .github/workflows/golangci-lint.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+name: golangci-lint
```
Comment (Xuanwo) [inline .github/workflows/golangci-lint.yml:1]: Please remove this action from this PR, as we will add it in #690

**Comment by Xuanwo** [comment]
Others LGTM

**Pull request merged**